srcpkgs/blanket/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'blanket'
 pkgname=blanket
-version=0.5.0
+version=0.6.0
 revision=1
 build_style=meson
 hostmakedepends="pkg-config gettext glib-devel"
-depends="gst1-python3 gst-plugins-bad1 gst-plugins-good1 libhandy1"
+depends="gst1-python3 gst-plugins-bad1 gst-plugins-good1 libhandy1 libadwaita"
```

Comment:
they moved from gtk3+libhandy to gtk4+libadwaita, remove libhandy from the dependency list

Reply:
I removed libhandy1, force pushed and re-tested. Everything seems to be working as expected.